docs/config.rst

```diff
@@ -175,6 +175,8 @@ disable sorting via
     with nengo.Network() as net:
         nengo_dl.configure_settings(sorter=noop_order_signals)
 
+.. _config-simplifications:
```

Comment:
This label doesn't appear to be used anywhere, is this a future-proofing thing or was there originally a link to this label that got removed?

Reply:
Ah yeah that used to be used but isn't (I had a whole section talking about how/when to turn on the `remove_reset_incs` optimization, but then figured out it was fine to just have it enabled by default).

Reply:
Coo, I'll remove and then this looks good to go :ok_hand: